SimeonInstaller.ps1

```diff
@@ -1605,12 +1631,14 @@ CRLFOption=CRLFAlways
                 variables = $pipelineVariables
             }
 
+            $pipelineId = $pipeline.id
             if ($pipeline) {
                 Write-Information "Pipeline '$pipelineName' already exists - updating"
 
                 $definition = irm @restProps "$apiBaseUrl/build/definitions/$($pipeline.id)?revision=$($pipeline.revision)" -Method Get
 
                 $body.variables = $definition.variables
+                if (!$Credential) { $body.variables | gm |? Name -in @('AadAuth:Username', 'AadAuth:Password') | % {    $body.PSObject.Properties.Remove($_.Name)   } }
```

Comment:
add 'AadAuth:TenantId' and 'AzureManagement:SubscriptionId' here...an alternative I thought of:
```
        if (!$UseServiceAccount) {
            $pipelineVariables['AadAuth:TenantId'] = @{
                allowOverride = $false
                value = $Tenant
            }
        }
```
set them to null here...your call